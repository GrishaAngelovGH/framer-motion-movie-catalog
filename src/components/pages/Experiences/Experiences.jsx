import { motion } from "framer-motion"

import LayoutPage from "components/pages/LayoutPage"
import Carousel from "components/Carousel"
import ProductCard from "components/ProductCard"

const Experiences = () => {
  const slides = [
    { image: "/images/hotdog-wedges.jpg", paragraphs: ["Enjoy a film feast", "Nothing beats watching a great film while tucking into an equally-great meal. Skip the restaurant, head to the cinema and order a delicious feast to take into the cinema. Try our chicken strips, nachos or new mozzarella sticks."] },
    { image: "/images/popcorn.jpg", paragraphs: ["Get the king of cinema snacks", "Popcorn and film, they go together like Batman and Robin or Woody and Buzz. That’s why we make sure ours is the best."] },
    { image: "/images/ice-cream.jpg", paragraphs: ["Treat yourself", "Fancy something indulgent? You’re spoiled for choice at the cinema. Order Ben &amp; Jerry’s Chocolate Fudge Brownie, Caramel Chew Chew or a dozen other flavours. Sip a cappuccino from Costa or mix it up with a sugar-free Tango Ice Blast."] }
  ]

  const favoriteProducts = [
    { image: "/images/kidscombo.jpg", title: "Kids Mix", content: "Get great value on snacks for your little ones with our Kids Mix combo, featuring a small popcorn, two snacks and a delicious drink for just £4.50." },
    { image: "/images/popcorn-and-coke.jpg", title: "Popcorn and Coca-Cola", content: "Going to the cinema wouldn’t be the same without popcorn. We make sure ours is perfectly popped for freshness. To wash it down, nothing beats an ice-cold, refreshing Coca-Cola. Order the dream team of cinema treats." },
    { image: "/images/ice-blast.jpg", title: "Tango Ice Blast", content: "Feel the frozen explosion with sugar-free Tango Ice Blast. It’s the guilt-free way to satisfy your sweet tooth. Order delicious slushy flavours like cherry, raspberry or a mix of both and let the icy zing dance on your tongue." }
  ]

  return (
    <LayoutPage>
      <div className="bg-blue-800 min-h-full flex flex-col items-center">
        <motion.img
          initial={{ y: "-100%" }}
          animate={{ y: "0%" }}
          transition={{ duration: 2 }}
          src="/images/hotdog-banner.png"
        />

        <motion.div
          initial={{ opacity: "0", display: "none" }}
          animate={{ opacity: "100%", display: "flex", flexDirection: "column", alignItems: "center" }}
          transition={{ duration: 1, delay: 3 }}
          className="w-3/4 md:w-1/2"
        >
          <h1 className="mt-10 mb-5 text-white text-center uppercase text-sm md:text-5xl">Bite-sized adventures</h1>
          <h1 className="m-10 text-white text-justify text-sm md:text-2xl">Turn your trip to the cinema into a cheerful dining experience with our selection of made-for-cinema bites. Treat yourself to nachos, hot dogs and chicken strips or share popcorn and chocolate-covered treats with your date.</h1>
          <Carousel className="mb-20" slides={slides} />
        </motion.div>

        <motion.div
          initial={{ opacity: "0" }}
          animate={{ opacity: "100%" }}
          transition={{ duration: 1, delay: 3 }}
          className="md:w-3/4 mb-20"
        >
          <h1 className="mb-10 text-white text-center uppercase text-sm md:text-5xl">Your Cinema Favorites</h1>

          <div className="flex flex-col xl:flex-row items-center gap-8">
            {
              favoriteProducts.map((v, i) => (
                <ProductCard key={i} {...v} />
              ))
            }
          </div>
        </motion.div>
      </div>
    </LayoutPage>
  )
}

export default Experiences