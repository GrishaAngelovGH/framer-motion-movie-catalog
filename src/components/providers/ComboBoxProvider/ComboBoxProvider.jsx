import React from "react"

const comboBoxes = {
  "12wer": { id: "12wer", title: "Combo Box #1", products: [{ image: "/images/combo-coca-cola.jpg", title: "Coca-Cola", price: 3 }, { image: "/images/combo-chips.jpg", title: "Chips", price: 7 }, { image: "/images/combo-ice-cream.jpg", title: "Ice-Cream", price: 10 }] },
  "23dfg": { id: "23dfg", title: "Combo Box #2", products: [{ image: "/images/combo-fanta.jpg", title: "Fanta", price: 3 }, { image: "/images/combo-pizza.jpg", title: "Pizza", price: 8 }, { image: "/images/combo-doughnut.jpg", title: "Doughnut", price: 4 }] },
  "34sjd": { id: "34sjd", title: "Combo Box #3", products: [{ image: "/images/combo-7up.jpg", title: "7Up", price: 3 }, { image: "/images/combo-popcorn.jpg", title: "Popcorn", price: 5 }, { image: "/images/combo-kitkat.jpg", title: "Kit Kat", price: 4 }] }
}

const ComboBoxProvider = ({ children }) => React.cloneElement(children, { comboBoxes })

export default ComboBoxProvider