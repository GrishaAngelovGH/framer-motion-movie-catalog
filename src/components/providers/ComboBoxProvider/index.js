import ComboBoxProvider from "./ComboBoxProvider"

export default ComboBoxProvider