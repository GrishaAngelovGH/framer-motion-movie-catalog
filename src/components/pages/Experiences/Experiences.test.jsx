import { render, screen, fireEvent } from "@testing-library/react"
import { BrowserRouter as Router, Routes, Route } from "react-router-dom"

import Experiences from "./Experiences"
import ComboBoxProvider from "components/providers/ComboBoxProvider"

test("should render pages/Experiences component", () => {
  const view = render(
    <Router>
      <Routes>
        <Route path="/" element={<ComboBoxProvider><Experiences /></ComboBoxProvider>} />
      </Routes>
    </Router>
  )

  expect(view).toMatchSnapshot()
})

test("should render pages/Experiences component with shopping cart button", () => {
  const view = render(
    <Router>
      <Routes>
        <Route path="/" element={<ComboBoxProvider><Experiences /></ComboBoxProvider>} />
      </Routes>
    </Router>
  )

  const AddToCartButton = screen.getAllByRole("button", { name: /Add To Cart/i })[0]
  fireEvent.click(AddToCartButton)

  expect(view).toMatchSnapshot()
  expect(screen.getByTestId("shopping-cart")).toBeInTheDocument()
})