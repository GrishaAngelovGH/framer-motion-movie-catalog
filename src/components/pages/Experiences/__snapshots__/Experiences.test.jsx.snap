// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render pages/Experiences component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex flex-col h-screen"
      >
        <div
          class="lg:h-[60px]"
        >
          <div
            class="flex flex-col md:flex-row justify-between items-center bg-blue-500 h-full p-2"
          >
            <a
              class="ms-5 flex items-center"
              href="/"
            >
              <img
                class="w-[40px]"
                src="/images/film-roll.png"
              />
              <h1
                class="ms-3 text-2xl font-bold text-blue-900"
              >
                Cinema
              </h1>
            </a>
            <div
              class="m-5 grid grid-rows-2 grid-cols-2 gap-2 md:grid-flow-col md:grid-rows-1"
            >
              <a
                class="text-white text-center hover:text-white hover:underline me-5 bg-blue-800 p-2 rounded-full"
                href="/"
              >
                Home
              </a>
              <a
                class="text-white text-center hover:text-white hover:underline me-5 bg-transparent p-2"
                href="/favorites"
              >
                Favorites
              </a>
              <a
                class="text-white text-center hover:text-white hover:underline me-5 bg-transparent p-2"
                href="/experiences"
              >
                Experiences
              </a>
            </div>
          </div>
        </div>
        <div
          class="h-full"
        >
          <div
            class="bg-blue-800 min-h-full flex flex-col items-center"
          >
            <img
              src="/images/hotdog-banner.png"
              style="transform: translateY(-100%) translateZ(0);"
            />
            <div
              class="w-3/4 md:w-1/2"
              style="opacity: 0; display: none;"
            >
              <h1
                class="mt-10 text-white text-center uppercase text-sm md:text-5xl"
              >
                Bite-sized adventures
              </h1>
              <h1
                class="m-5 text-white text-justify text-sm md:text-2xl"
              >
                Turn your trip to the cinema into a cheerful dining experience with our selection of made-for-cinema bites. Treat yourself to nachos, hot dogs and chicken strips or share popcorn and chocolate-covered treats with your date.
              </h1>
              <div
                class="carousel-root mb-10"
                tabindex="0"
              >
                <div
                  class="carousel carousel-slider"
                  style="width: 100%;"
                >
                  <ul
                    class="control-dots"
                  >
                    <li
                      aria-label="slide item 1"
                      class="dot selected"
                      role="button"
                      tabindex="0"
                      value="0"
                    />
                    <li
                      aria-label="slide item 2"
                      class="dot"
                      role="button"
                      tabindex="0"
                      value="1"
                    />
                    <li
                      aria-label="slide item 3"
                      class="dot"
                      role="button"
                      tabindex="0"
                      value="2"
                    />
                  </ul>
                  <button
                    aria-label="previous slide / item"
                    class="control-arrow control-prev"
                    type="button"
                  />
                  <div
                    class="slider-wrapper axis-horizontal"
                  >
                    <ul
                      class="slider animated"
                      style="transform: translate3d(-100%,0,0); transition-duration: 350ms;"
                    >
                      <li
                        class="slide"
                      >
                        <div>
                          <img
                            src="/images/ice-cream.jpg"
                          />
                          <div
                            class="legend hidden md:block"
                          >
                            <p>
                              Treat yourself
                            </p>
                            <p>
                              Fancy something indulgent? You’re spoiled for choice at the cinema. Order Ben & Jerry’s Chocolate Fudge Brownie, Caramel Chew Chew or a dozen other flavours. Sip a cappuccino from Costa or mix it up with a sugar-free Tango Ice Blast.
                            </p>
                          </div>
                        </div>
                      </li>
                      <li
                        class="slide selected previous"
                      >
                        <div>
                          <img
                            src="/images/hotdog-wedges.jpg"
                          />
                          <div
                            class="legend hidden md:block"
                          >
                            <p>
                              Enjoy a film feast
                            </p>
                            <p>
                              Nothing beats watching a great film while tucking into an equally-great meal. Skip the restaurant, head to the cinema and order a delicious feast to take into the cinema. Try our chicken strips, nachos or new mozzarella sticks.
                            </p>
                          </div>
                        </div>
                      </li>
                      <li
                        class="slide"
                      >
                        <div>
                          <img
                            src="/images/popcorn.jpg"
                          />
                          <div
                            class="legend hidden md:block"
                          >
                            <p>
                              Get the king of cinema snacks
                            </p>
                            <p>
                              Popcorn and film, they go together like Batman and Robin or Woody and Buzz. That’s why we make sure ours is the best.
                            </p>
                          </div>
                        </div>
                      </li>
                      <li
                        class="slide"
                      >
                        <div>
                          <img
                            src="/images/ice-cream.jpg"
                          />
                          <div
                            class="legend hidden md:block"
                          >
                            <p>
                              Treat yourself
                            </p>
                            <p>
                              Fancy something indulgent? You’re spoiled for choice at the cinema. Order Ben & Jerry’s Chocolate Fudge Brownie, Caramel Chew Chew or a dozen other flavours. Sip a cappuccino from Costa or mix it up with a sugar-free Tango Ice Blast.
                            </p>
                          </div>
                        </div>
                      </li>
                      <li
                        class="slide selected previous"
                      >
                        <div>
                          <img
                            src="/images/hotdog-wedges.jpg"
                          />
                          <div
                            class="legend hidden md:block"
                          >
                            <p>
                              Enjoy a film feast
                            </p>
                            <p>
                              Nothing beats watching a great film while tucking into an equally-great meal. Skip the restaurant, head to the cinema and order a delicious feast to take into the cinema. Try our chicken strips, nachos or new mozzarella sticks.
                            </p>
                          </div>
                        </div>
                      </li>
                    </ul>
                  </div>
                  <button
                    aria-label="next slide / item"
                    class="control-arrow control-next"
                    type="button"
                  />
                  <p
                    class="carousel-status"
                  >
                    1 of 3
                  </p>
                </div>
              </div>
            </div>
            <div
              class="md:w-3/4 mb-5"
              style="opacity: 0;"
            >
              <h1
                class="mb-10 text-white text-center uppercase text-sm md:text-5xl"
              >
                Your Cinema Favorites
              </h1>
              <div
                class="flex flex-col xl:flex-row items-center gap-8"
              >
                <div
                  class="min-w-[300px] bg-white p-2 min-h-[400px] rounded-md w-2/3 text-center"
                >
                  <img
                    class="inline-block w-full rounded-md"
                    src="/images/kidscombo.jpg"
                  />
                  <p
                    class="m-5 text-xl"
                  >
                    Kids Mix
                  </p>
                  <p
                    class="text-gray-500"
                  >
                    Get great value on snacks for your little ones with our Kids Mix combo, featuring a small popcorn, two snacks and a delicious drink for just £4.50.
                  </p>
                </div>
                <div
                  class="min-w-[300px] bg-white p-2 min-h-[400px] rounded-md w-2/3 text-center"
                >
                  <img
                    class="inline-block w-full rounded-md"
                    src="/images/popcorn-and-coke.jpg"
                  />
                  <p
                    class="m-5 text-xl"
                  >
                    Popcorn and Coca-Cola
                  </p>
                  <p
                    class="text-gray-500"
                  >
                    Going to the cinema wouldn’t be the same without popcorn. We make sure ours is perfectly popped for freshness. To wash it down, nothing beats an ice-cold, refreshing Coca-Cola. Order the dream team of cinema treats.
                  </p>
                </div>
                <div
                  class="min-w-[300px] bg-white p-2 min-h-[400px] rounded-md w-2/3 text-center"
                >
                  <img
                    class="inline-block w-full rounded-md"
                    src="/images/ice-blast.jpg"
                  />
                  <p
                    class="m-5 text-xl"
                  >
                    Tango Ice Blast
                  </p>
                  <p
                    class="text-gray-500"
                  >
                    Feel the frozen explosion with sugar-free Tango Ice Blast. It’s the guilt-free way to satisfy your sweet tooth. Order delicious slushy flavours like cherry, raspberry or a mix of both and let the icy zing dance on your tongue.
                  </p>
                </div>
              </div>
            </div>
            <div
              class="m-5 mb-10 md:w-1/2"
              style="opacity: 0;"
            >
              <h1
                class="text-sm md:text-5xl text-white text-center uppercase"
              >
                Try Our Combo Boxes
              </h1>
              <div
                class="bg-white rounded-md mt-10 text-center min-w-[300px] relative"
              >
                <div
                  class="text-white  p-1 bg-blue-400 rounded-t"
                >
                  Combo Box #1
                </div>
                <div
                  class="flex flex-col md:flex-row justify-evenly p-5"
                >
                  <div
                    class="m-3"
                  >
                    <img
                      class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                      src="/images/combo-coca-cola.jpg"
                    />
                    <div
                      class="mt-2 text-gray-500"
                    >
                      Coca-Cola
                       (
                      3
                      £)
                    </div>
                  </div>
                  <div
                    class="m-3"
                  >
                    <img
                      class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                      src="/images/combo-chips.jpg"
                    />
                    <div
                      class="mt-2 text-gray-500"
                    >
                      Chips
                       (
                      7
                      £)
                    </div>
                  </div>
                  <div
                    class="m-3"
                  >
                    <img
                      class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                      src="/images/combo-ice-cream.jpg"
                    />
                    <div
                      class="mt-2 text-gray-500"
                    >
                      Ice-Cream
                       (
                      10
                      £)
                    </div>
                  </div>
                </div>
                <p
                  class="text-xl"
                >
                  Total: 
                  20
                  £
                </p>
                <button
                  class="bg-blue-500 hover:bg-blue-700 text-white rounded-full m-3 visible"
                  role="button"
                >
                  Add To Cart
                </button>
              </div>
              <div
                class="bg-white rounded-md mt-10 text-center min-w-[300px] relative"
              >
                <div
                  class="text-white  p-1 bg-blue-400 rounded-t"
                >
                  Combo Box #2
                </div>
                <div
                  class="flex flex-col md:flex-row justify-evenly p-5"
                >
                  <div
                    class="m-3"
                  >
                    <img
                      class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                      src="/images/combo-fanta.jpg"
                    />
                    <div
                      class="mt-2 text-gray-500"
                    >
                      Fanta
                       (
                      3
                      £)
                    </div>
                  </div>
                  <div
                    class="m-3"
                  >
                    <img
                      class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                      src="/images/combo-pizza.jpg"
                    />
                    <div
                      class="mt-2 text-gray-500"
                    >
                      Pizza
                       (
                      8
                      £)
                    </div>
                  </div>
                  <div
                    class="m-3"
                  >
                    <img
                      class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                      src="/images/combo-doughnut.jpg"
                    />
                    <div
                      class="mt-2 text-gray-500"
                    >
                      Doughnut
                       (
                      4
                      £)
                    </div>
                  </div>
                </div>
                <p
                  class="text-xl"
                >
                  Total: 
                  15
                  £
                </p>
                <button
                  class="bg-blue-500 hover:bg-blue-700 text-white rounded-full m-3 visible"
                  role="button"
                >
                  Add To Cart
                </button>
              </div>
              <div
                class="bg-white rounded-md mt-10 text-center min-w-[300px] relative"
              >
                <div
                  class="text-white  p-1 bg-blue-400 rounded-t"
                >
                  Combo Box #3
                </div>
                <div
                  class="flex flex-col md:flex-row justify-evenly p-5"
                >
                  <div
                    class="m-3"
                  >
                    <img
                      class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                      src="/images/combo-7up.jpg"
                    />
                    <div
                      class="mt-2 text-gray-500"
                    >
                      7Up
                       (
                      3
                      £)
                    </div>
                  </div>
                  <div
                    class="m-3"
                  >
                    <img
                      class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                      src="/images/combo-popcorn.jpg"
                    />
                    <div
                      class="mt-2 text-gray-500"
                    >
                      Popcorn
                       (
                      5
                      £)
                    </div>
                  </div>
                  <div
                    class="m-3"
                  >
                    <img
                      class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                      src="/images/combo-kitkat.jpg"
                    />
                    <div
                      class="mt-2 text-gray-500"
                    >
                      Kit Kat
                       (
                      4
                      £)
                    </div>
                  </div>
                </div>
                <p
                  class="text-xl"
                >
                  Total: 
                  12
                  £
                </p>
                <button
                  class="bg-blue-500 hover:bg-blue-700 text-white rounded-full m-3 visible"
                  role="button"
                >
                  Add To Cart
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex flex-col h-screen"
    >
      <div
        class="lg:h-[60px]"
      >
        <div
          class="flex flex-col md:flex-row justify-between items-center bg-blue-500 h-full p-2"
        >
          <a
            class="ms-5 flex items-center"
            href="/"
          >
            <img
              class="w-[40px]"
              src="/images/film-roll.png"
            />
            <h1
              class="ms-3 text-2xl font-bold text-blue-900"
            >
              Cinema
            </h1>
          </a>
          <div
            class="m-5 grid grid-rows-2 grid-cols-2 gap-2 md:grid-flow-col md:grid-rows-1"
          >
            <a
              class="text-white text-center hover:text-white hover:underline me-5 bg-blue-800 p-2 rounded-full"
              href="/"
            >
              Home
            </a>
            <a
              class="text-white text-center hover:text-white hover:underline me-5 bg-transparent p-2"
              href="/favorites"
            >
              Favorites
            </a>
            <a
              class="text-white text-center hover:text-white hover:underline me-5 bg-transparent p-2"
              href="/experiences"
            >
              Experiences
            </a>
          </div>
        </div>
      </div>
      <div
        class="h-full"
      >
        <div
          class="bg-blue-800 min-h-full flex flex-col items-center"
        >
          <img
            src="/images/hotdog-banner.png"
            style="transform: translateY(-100%) translateZ(0);"
          />
          <div
            class="w-3/4 md:w-1/2"
            style="opacity: 0; display: none;"
          >
            <h1
              class="mt-10 text-white text-center uppercase text-sm md:text-5xl"
            >
              Bite-sized adventures
            </h1>
            <h1
              class="m-5 text-white text-justify text-sm md:text-2xl"
            >
              Turn your trip to the cinema into a cheerful dining experience with our selection of made-for-cinema bites. Treat yourself to nachos, hot dogs and chicken strips or share popcorn and chocolate-covered treats with your date.
            </h1>
            <div
              class="carousel-root mb-10"
              tabindex="0"
            >
              <div
                class="carousel carousel-slider"
                style="width: 100%;"
              >
                <ul
                  class="control-dots"
                >
                  <li
                    aria-label="slide item 1"
                    class="dot selected"
                    role="button"
                    tabindex="0"
                    value="0"
                  />
                  <li
                    aria-label="slide item 2"
                    class="dot"
                    role="button"
                    tabindex="0"
                    value="1"
                  />
                  <li
                    aria-label="slide item 3"
                    class="dot"
                    role="button"
                    tabindex="0"
                    value="2"
                  />
                </ul>
                <button
                  aria-label="previous slide / item"
                  class="control-arrow control-prev"
                  type="button"
                />
                <div
                  class="slider-wrapper axis-horizontal"
                >
                  <ul
                    class="slider animated"
                    style="transform: translate3d(-100%,0,0); transition-duration: 350ms;"
                  >
                    <li
                      class="slide"
                    >
                      <div>
                        <img
                          src="/images/ice-cream.jpg"
                        />
                        <div
                          class="legend hidden md:block"
                        >
                          <p>
                            Treat yourself
                          </p>
                          <p>
                            Fancy something indulgent? You’re spoiled for choice at the cinema. Order Ben & Jerry’s Chocolate Fudge Brownie, Caramel Chew Chew or a dozen other flavours. Sip a cappuccino from Costa or mix it up with a sugar-free Tango Ice Blast.
                          </p>
                        </div>
                      </div>
                    </li>
                    <li
                      class="slide selected previous"
                    >
                      <div>
                        <img
                          src="/images/hotdog-wedges.jpg"
                        />
                        <div
                          class="legend hidden md:block"
                        >
                          <p>
                            Enjoy a film feast
                          </p>
                          <p>
                            Nothing beats watching a great film while tucking into an equally-great meal. Skip the restaurant, head to the cinema and order a delicious feast to take into the cinema. Try our chicken strips, nachos or new mozzarella sticks.
                          </p>
                        </div>
                      </div>
                    </li>
                    <li
                      class="slide"
                    >
                      <div>
                        <img
                          src="/images/popcorn.jpg"
                        />
                        <div
                          class="legend hidden md:block"
                        >
                          <p>
                            Get the king of cinema snacks
                          </p>
                          <p>
                            Popcorn and film, they go together like Batman and Robin or Woody and Buzz. That’s why we make sure ours is the best.
                          </p>
                        </div>
                      </div>
                    </li>
                    <li
                      class="slide"
                    >
                      <div>
                        <img
                          src="/images/ice-cream.jpg"
                        />
                        <div
                          class="legend hidden md:block"
                        >
                          <p>
                            Treat yourself
                          </p>
                          <p>
                            Fancy something indulgent? You’re spoiled for choice at the cinema. Order Ben & Jerry’s Chocolate Fudge Brownie, Caramel Chew Chew or a dozen other flavours. Sip a cappuccino from Costa or mix it up with a sugar-free Tango Ice Blast.
                          </p>
                        </div>
                      </div>
                    </li>
                    <li
                      class="slide selected previous"
                    >
                      <div>
                        <img
                          src="/images/hotdog-wedges.jpg"
                        />
                        <div
                          class="legend hidden md:block"
                        >
                          <p>
                            Enjoy a film feast
                          </p>
                          <p>
                            Nothing beats watching a great film while tucking into an equally-great meal. Skip the restaurant, head to the cinema and order a delicious feast to take into the cinema. Try our chicken strips, nachos or new mozzarella sticks.
                          </p>
                        </div>
                      </div>
                    </li>
                  </ul>
                </div>
                <button
                  aria-label="next slide / item"
                  class="control-arrow control-next"
                  type="button"
                />
                <p
                  class="carousel-status"
                >
                  1 of 3
                </p>
              </div>
            </div>
          </div>
          <div
            class="md:w-3/4 mb-5"
            style="opacity: 0;"
          >
            <h1
              class="mb-10 text-white text-center uppercase text-sm md:text-5xl"
            >
              Your Cinema Favorites
            </h1>
            <div
              class="flex flex-col xl:flex-row items-center gap-8"
            >
              <div
                class="min-w-[300px] bg-white p-2 min-h-[400px] rounded-md w-2/3 text-center"
              >
                <img
                  class="inline-block w-full rounded-md"
                  src="/images/kidscombo.jpg"
                />
                <p
                  class="m-5 text-xl"
                >
                  Kids Mix
                </p>
                <p
                  class="text-gray-500"
                >
                  Get great value on snacks for your little ones with our Kids Mix combo, featuring a small popcorn, two snacks and a delicious drink for just £4.50.
                </p>
              </div>
              <div
                class="min-w-[300px] bg-white p-2 min-h-[400px] rounded-md w-2/3 text-center"
              >
                <img
                  class="inline-block w-full rounded-md"
                  src="/images/popcorn-and-coke.jpg"
                />
                <p
                  class="m-5 text-xl"
                >
                  Popcorn and Coca-Cola
                </p>
                <p
                  class="text-gray-500"
                >
                  Going to the cinema wouldn’t be the same without popcorn. We make sure ours is perfectly popped for freshness. To wash it down, nothing beats an ice-cold, refreshing Coca-Cola. Order the dream team of cinema treats.
                </p>
              </div>
              <div
                class="min-w-[300px] bg-white p-2 min-h-[400px] rounded-md w-2/3 text-center"
              >
                <img
                  class="inline-block w-full rounded-md"
                  src="/images/ice-blast.jpg"
                />
                <p
                  class="m-5 text-xl"
                >
                  Tango Ice Blast
                </p>
                <p
                  class="text-gray-500"
                >
                  Feel the frozen explosion with sugar-free Tango Ice Blast. It’s the guilt-free way to satisfy your sweet tooth. Order delicious slushy flavours like cherry, raspberry or a mix of both and let the icy zing dance on your tongue.
                </p>
              </div>
            </div>
          </div>
          <div
            class="m-5 mb-10 md:w-1/2"
            style="opacity: 0;"
          >
            <h1
              class="text-sm md:text-5xl text-white text-center uppercase"
            >
              Try Our Combo Boxes
            </h1>
            <div
              class="bg-white rounded-md mt-10 text-center min-w-[300px] relative"
            >
              <div
                class="text-white  p-1 bg-blue-400 rounded-t"
              >
                Combo Box #1
              </div>
              <div
                class="flex flex-col md:flex-row justify-evenly p-5"
              >
                <div
                  class="m-3"
                >
                  <img
                    class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                    src="/images/combo-coca-cola.jpg"
                  />
                  <div
                    class="mt-2 text-gray-500"
                  >
                    Coca-Cola
                     (
                    3
                    £)
                  </div>
                </div>
                <div
                  class="m-3"
                >
                  <img
                    class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                    src="/images/combo-chips.jpg"
                  />
                  <div
                    class="mt-2 text-gray-500"
                  >
                    Chips
                     (
                    7
                    £)
                  </div>
                </div>
                <div
                  class="m-3"
                >
                  <img
                    class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                    src="/images/combo-ice-cream.jpg"
                  />
                  <div
                    class="mt-2 text-gray-500"
                  >
                    Ice-Cream
                     (
                    10
                    £)
                  </div>
                </div>
              </div>
              <p
                class="text-xl"
              >
                Total: 
                20
                £
              </p>
              <button
                class="bg-blue-500 hover:bg-blue-700 text-white rounded-full m-3 visible"
                role="button"
              >
                Add To Cart
              </button>
            </div>
            <div
              class="bg-white rounded-md mt-10 text-center min-w-[300px] relative"
            >
              <div
                class="text-white  p-1 bg-blue-400 rounded-t"
              >
                Combo Box #2
              </div>
              <div
                class="flex flex-col md:flex-row justify-evenly p-5"
              >
                <div
                  class="m-3"
                >
                  <img
                    class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                    src="/images/combo-fanta.jpg"
                  />
                  <div
                    class="mt-2 text-gray-500"
                  >
                    Fanta
                     (
                    3
                    £)
                  </div>
                </div>
                <div
                  class="m-3"
                >
                  <img
                    class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                    src="/images/combo-pizza.jpg"
                  />
                  <div
                    class="mt-2 text-gray-500"
                  >
                    Pizza
                     (
                    8
                    £)
                  </div>
                </div>
                <div
                  class="m-3"
                >
                  <img
                    class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                    src="/images/combo-doughnut.jpg"
                  />
                  <div
                    class="mt-2 text-gray-500"
                  >
                    Doughnut
                     (
                    4
                    £)
                  </div>
                </div>
              </div>
              <p
                class="text-xl"
              >
                Total: 
                15
                £
              </p>
              <button
                class="bg-blue-500 hover:bg-blue-700 text-white rounded-full m-3 visible"
                role="button"
              >
                Add To Cart
              </button>
            </div>
            <div
              class="bg-white rounded-md mt-10 text-center min-w-[300px] relative"
            >
              <div
                class="text-white  p-1 bg-blue-400 rounded-t"
              >
                Combo Box #3
              </div>
              <div
                class="flex flex-col md:flex-row justify-evenly p-5"
              >
                <div
                  class="m-3"
                >
                  <img
                    class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                    src="/images/combo-7up.jpg"
                  />
                  <div
                    class="mt-2 text-gray-500"
                  >
                    7Up
                     (
                    3
                    £)
                  </div>
                </div>
                <div
                  class="m-3"
                >
                  <img
                    class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                    src="/images/combo-popcorn.jpg"
                  />
                  <div
                    class="mt-2 text-gray-500"
                  >
                    Popcorn
                     (
                    5
                    £)
                  </div>
                </div>
                <div
                  class="m-3"
                >
                  <img
                    class="inline w-[100px] h-[100px] border border-gray-600 rounded-full p-2"
                    src="/images/combo-kitkat.jpg"
                  />
                  <div
                    class="mt-2 text-gray-500"
                  >
                    Kit Kat
                     (
                    4
                    £)
                  </div>
                </div>
              </div>
              <p
                class="text-xl"
              >
                Total: 
                12
                £
              </p>
              <button
                class="bg-blue-500 hover:bg-blue-700 text-white rounded-full m-3 visible"
                role="button"
              >
                Add To Cart
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
